sp/mk/sp_dev_kit.mk

```diff
@@ -0,0 +1,115 @@
+# Get the dir of the sp-dev-kit, requires make version 3.81 or later
+sp-dev-kit-dir := $(patsubst %/,%,$(abspath $(dir $(lastword $(MAKEFILE_LIST)))..))
+
+.PHONY: all
+all:
+
+include $(sp-dev-kit-dir)/mk/conf.mk
+sp-dev-kit-dir$(sm) := $(sp-dev-kit-dir)
+
+ifneq (1, $(words $(BINARY)))
+$(error You must specify the BINARY variable)
+endif
+
+ifneq ($O,)
+out-dir := $O
+else
+out-dir := .
+endif
+link-out-dir := $(out-dir)	# backward compat
```

Comment:
i think you don't need this legacy $(link-out-dir).

Reply:
Done